src/sentry/static/sentry/app/components/avatar/sentryAppAvatar.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import BaseAvatar from 'app/components/avatar/baseAvatar';
+import SentryTypes from 'app/sentryTypes';
+
+class SentryAppAvatar extends React.Component {
```

Comment:
@EvanPurkhiser brought this up but is there a better name for this so it doesn't clash with https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/index.js#L81 ?

A valid solution may be to just rename that instance of `SentryApp`

Reply:
I think we should probably just rename our global. `SentryApp` is all over the place in the backend as well.

Reply:
I named it `SentryApplication` for now